src/main/java/com/cgi/ipsen5/Config/SecurityConfig.java

```diff
@@ -62,9 +62,9 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                         .requestMatchers("/api/v1/auth/**").permitAll()
                         .requestMatchers("/api/v1/user/register").permitAll()
                         .requestMatchers("/api/v1/user/reset-password").permitAll()
-                        .requestMatchers("/api/v1/user").hasAuthority("ADMIN")
-                        .requestMatchers("/api/v1/user/{id}/get-favorite-colleagues").hasAuthority("USER")
-                        .requestMatchers("/api/v1/user/{id}/add-favorite-colleague").hasAuthority("USER")
+                        .requestMatchers("/api/v1/user/favorite-colleagues").authenticated()
+                        .requestMatchers("/api/v1/user/standard-location").authenticated()
+                        .requestMatchers("/api/v1/user").authenticated()
```

Comment:
is .authenticated nodig? Want die is later toegevoegd aan het einde na .anyRequest()

Reply:
oh lol klopt